### PR TITLE
Last week in ManageIQ 2016-08-16

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

File: source/blog/2016-08-15-last-week-in-manageiq.html.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+reports editor controller which was already implemented elsewhere. This is now removed!
+
+## Wrapping up
+
+There were way too many PRs to cover here in detail. Feel free to
+check them all out [here][PRs merged last week]. Look out for the next
+installment of *Last Week in ManageIQ*!
+
+[PRs merged last week]: https://github.com/ManageIQ/manageiq/pulls?page=1&q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222016-08-08+..+2016-08-14%22+sort%3Acreated-desc&utf8=%E2%9C%93
+[Commits merged last week]: https://github.com/manageiq/manageiq/compare/master@%7B2016-08-08%7D...@%7B2016-08-14%7D
```
Comment (syncrou) [inline source/blog/2016-08-15-last-week-in-manageiq.html.md]: @skateman - It looks like this isn't being used anywhere.

File: source/blog/2016-08-15-last-week-in-manageiq.html.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+title: "Last Week in ManageIQ: About modal, gettext improvements, refactoring and more!"
+date: 2016-08-15 06:20 UTC
+author: skateman
+comments: true
+published: true
+---
+
+Hi! [David](https://twitter.com/halaszdavid) here, from the UI team!
+
+Let's take a quick review about the changes to ManageIQ since last week.
```
Comment (imtayadeway) [inline source/blog/2016-08-15-last-week-in-manageiq.html.md]: s/about/of/

File: source/blog/2016-08-15-last-week-in-manageiq.html.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+## Featured
+
+The last week [114 merged pull requests][PRs merged last week] have been
```
Comment (imtayadeway) [inline source/blog/2016-08-15-last-week-in-manageiq.html.md]: Possibly "In the last week" or "The last week has seen [...] pull requests produced"?

**Comment by imtayadeway** [comment]
@skateman this looks fab! Just a couple of minor comments above :blue_heart:

**Comment by skateman** [comment]
@imtayadeway all fixed!

File: source/blog/2016-08-15-last-week-in-manageiq.html.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+We are constantly improving the networking support in ManageIQ. This PR
+is the first step to have support for Load Balancers available in cloud
+environments.
+
+### [Add 'About' Modal](https://github.com/ManageIQ/manageiq/pull/9989)
+
+Thanks to the improvements in the PatternFly project, we are now able to
+display a fancy about modal. This displays information about the browser
+session and the used version number of ManageIQ.
```
Comment (syncrou) [inline source/blog/2016-08-15-last-week-in-manageiq.html.md]: @skateman - One very small request 😄  

 I believe the `,` isn't needed on line 62.

File: source/blog/2016-08-15-last-week-in-manageiq.html.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+Hi! [David](https://twitter.com/halaszdavid) here, from the UI team!
+
+Let's take a quick review of the changes to ManageIQ since last week.
+
+## Featured
+
+In the last week [114 merged pull requests][PRs merged last week] have been
+produced by 42 of our contributors. We pretty much like this round number
```
Comment (chrisarcand) [inline source/blog/2016-08-15-last-week-in-manageiq.html.md]: s/pretty/very/ ?

File: source/blog/2016-08-15-last-week-in-manageiq.html.md (hunk 4 above)
Comment (skateman) [inline source/blog/2016-08-15-last-week-in-manageiq.html.md]: fixed

File: source/blog/2016-08-15-last-week-in-manageiq.html.md (hunk 5 above)
Comment (skateman) [inline source/blog/2016-08-15-last-week-in-manageiq.html.md]: thanks, fixed

**Comment by syncrou** [comment]
👍

**Comment by JPrause** [comment]
@chessbyte it appears we are clear to merge. :+1: 

Thanks @syncrou, @chrisarcand, and @imtayadeway for your reviews!

**Pull request merged**